pcsx2/Frontend/XInputSource.h

```diff
@@ -21,6 +21,39 @@
 #include <mutex>
 #include <vector>
 
+// SCP XInput extension
+typedef struct
```

Comment:
This could be forward declared in the header, with the actual struct in the cpp file.